iot_relay_server/app/ai_agent.py

```
@@ -54,237 +63,273 @@
 }
 
 
-# ─── 규칙 기반 판단 ───
+# ─── 규칙 기반 판단 (긴급 대응 — 30초마다 실행) ───
 
 def _apply_rules(sensor_data: dict, weather: dict) -> list[dict]:
```

Ruff (0.15.9)

[warning] Too many branches (16 > 12)

(PLR0912)